src/components/DateRangePickerInput.jsx

```diff
@@ -297,11 +316,11 @@ function DateRangePickerInput({
             styles.DateRangePickerInput_clearDates,
             small && styles.DateRangePickerInput_clearDates__small,
             !customCloseIcon && styles.DateRangePickerInput_clearDates_default,
-            !(startDate || endDate) && styles.DateRangePickerInput_clearDates__hide,
+            !(startDate || endDate) &&
```

Comment:
Same thing here - do we need to implement our own UI to call the clear function for custom rendering?